logs_http.module

```diff
@@ -152,6 +146,27 @@ function logs_http_register_event(array $log_entry) {
   // Remove empty values, to prevent errors in the indexing of the JSON.
   $event = logs_http_array_remove_empty($event);
 
+  if (variable_get('logs_http_debugging_mode', FALSE)) {
+    if (!isset($events['debugging'])) {
+      $debugging_event = $event;
+      unset($debugging_event['type']);
```

Comment:
Because there is quite a lot of extra items in the $event array that you're adding at the beginning of logs_http_register_event() which we're including here as well so that it is possible to create a link between the real event and the debugging information. Imagine you're having a site with a lot of traffic, the log could end up with lot's of events in parallel from different sessions and then it is important which debugging information belongs to which request.